### PR TITLE
Allow exempting elytras

### PR DESCRIPTION
**Pull request by ManInMyVan**
https://discord.com/channels/811396969670901800/1267889161629864101/1283298260626182226

### CODE REVIEW

File: src/main/java/ac/grim/grimac/predictionengine/MovementCheckRunner.java [hunk 1]
```diff
@@ -429,7 +429,7 @@ private void check(PositionUpdate update) {
             // Dead players can't cheat, if you find a way how they could, open an issue
             player.predictedVelocity = new VectorData(new Vector(), VectorData.VectorType.Dead);
             player.clientVelocity = new Vector();
-        } else if (player.disableGrim || (PacketEvents.getAPI().getServerManager().getVersion().isNewerThanOrEquals(ServerVersion.V_1_8) && player.gamemode == GameMode.SPECTATOR) || player.isFlying) {
+        } else if (player.disableGrim || (PacketEvents.getAPI().getServerManager().getVersion().isNewerThanOrEquals(ServerVersion.V_1_8) && player.gamemode == GameMode.SPECTATOR) || player.isFlying || player.isExemptElytra() && player.isGliding) {
```
Comment (SamB440) [inline src/main/java/ac/grim/grimac/predictionengine/MovementCheckRunner.java]: Please add parenthesis to the new and statement to make it clear

**Review by SamB440 — CHANGES_REQUESTED** [review]

**Comment by ManInMyVan** [comment]
If anyone's wondering why it isn't in the config.yml, I don't want people to enable it because of falses, especially without reporting them

**Review by SamB440 — APPROVED** [review]

**Comment by ghost** [comment]
Can't read that discord message but if this change is to "patch" false flags when using elytra then I'm not a fan of this

**Comment by ManInMyVan** [comment]
> Can't read that discord message but if this change is to "patch" false flags when using elytra then I'm not a fan of this

anarchy server wanted to allow elytra fly

**Comment by ghost** [comment]
> > Can't read that discord message but if this change is to "patch" false flags when using elytra then I'm not a fan of this
> 
> anarchy server wanted to allow elytra fly

ah okay

**Pull request merged**